### PR TITLE
Clarify Readme

### PR DESCRIPTION
**Pull request by bobbysteel**

### CODE REVIEW

**Comment by bobbysteel** [comment]
cool

File: README.md [hunk 1]
```diff
@@ -103,45 +120,54 @@ Once you have the browser instance, you can call the methods to interact with it
...
+  // Check the select current value
+  const myCurrentSubscriptionPlan = await browser.getValue('#subscriptionSelect')
+  console.log(myCurrentSubscriptionPlan) // {type: 'string', value: '1 month' }
+
+  // Edit the subscription
+  await browser.select('#subscriptionSelect', '3 months')
+  await browser.click('#Save')
+  
+  // Take a screenshot
+  const rdata = await browser.getScreenshot()
```
Comment (LucianoGanga) [inline README.md]: You can also make use of the . saveScreenshot() method, so you don't need to handle the file saving :)

**Comment by bobbysteel** [comment]
Changes done :)

**Comment by LucianoGanga** [comment]
Thanks @bobbysteel ! :D

**Comment by LucianoGanga** [comment]
I'll publish the changes in the NPM package in the next release, as the readme changes doesn't affect the code

**Pull request merged**